### ISSUE TITLE
could not determ Windows OS version. 

### ISSUE THREAD

**Issue by MandiYang** [winegui/WineGUI#1]
Creating a Windows10 (32 and 64)bit didn't work
![rfrf](https://user-images.githubusercontent.com/64307609/111863997-e2353100-895e-11eb-9a21-5387252d59ae.png)
![nnn](https://user-images.githubusercontent.com/64307609/111864025-ed885c80-895e-11eb-8f2f-d9c76e1c502b.png)
![rfrfff](https://user-images.githubusercontent.com/64307609/111864031-f5480100-895e-11eb-9403-9bfe869ced67.png)

```
mandi@rez-latitude:~$ '/home/mandi/WineGui-master/build/bin/winegui' 
Catched Failed to open file “”: No such file or directory
0034:fixme:font:get_name_record_codepage encoding 20 not handled, platform 1.
0034:fixme:font:get_name_record_codepage encoding 20 not handled, platform 1.
0064:fixme:font:get_name_record_codepage encoding 20 not handled, platform 1.
0064:fixme:font:get_name_record_codepage encoding 20 not handled, platform 1.
006c:fixme:font:get_name_record_codepage encoding 20 not handled, platform 1.
006c:fixme:font:get_name_record_codepage encoding 20 not handled, platform 1.
INTEL-MESA: warning: Performance support disabled, consider sysctl dev.i915.perf_stream_paranoid=0

002c:fixme:font:get_name_record_codepage encoding 20 not handled, platform 1.
002c:fixme:font:get_name_record_codepage encoding 20 not handled, platform 1.
00f4:fixme:font:get_name_record_codepage encoding 20 not handled, platform 1.
00f4:fixme:font:get_name_record_codepage encoding 20 not handled, platform 1.
00fc:fixme:font:get_name_record_codepage encoding 20 not handled, platform 1.
00fc:fixme:font:get_name_record_codepage encoding 20 not handled, platform 1.
```

Platform: Ubuntu 20.04
wine version: 6.0

**Comment by MandiYang** [comment]
But creating (32bit and 64bit) Windows XP work
![Screenshot from 2021-03-20 09-35-05](https://user-images.githubusercontent.com/64307609/111864150-a64e9b80-895f-11eb-92d2-023e60411630.png)
 
Worked

**Comment by melroy89** [comment]
Oké, thank you so much. It's only a problem with creating a windows 10 wine bottle. I will check the code.

**Comment by MandiYang** [comment]
It happends the same problem when creating windows 8 machine (64 and 32 bit)

**Comment by melroy89** [comment]
https://gitlab.melroy.org/melroy/winegui/-/issues/23

I was indeed able to reproduce this problem (also with Wine version 5.0):

![image](https://user-images.githubusercontent.com/628926/112154416-8d0a5100-8be4-11eb-93f9-16059c7c0507.png)

**Comment by melroy89** [comment]
So it has to do with the Build NT number mismatch (eg. 17134 vs 10240 build number), I will fix this. And also implement a fallback system, so it will not happen that often anymore in the future. (hopefully :) ).

**Comment by melroy89** [comment]
It's all fixed now, see release v1.5.2: https://gitlab.melroy.org/melroy/winegui/-/releases 🚀 

You could [download for example the .deb package](https://winegui.melroy.org/downloads/WineGUI-v1.5.2.deb) (Debian/Ubuntu distros and related).

**Comment by melroy89** [comment]
I recently released even better versions of WineGUI. Check it out: https://gitlab.melroy.org/melroy/winegui/-/releases
